var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -20,61 +22,84 @@ class Visit(CMakePackage):
     version('2.10.1', '3cbca162fdb0249f17c4456605c4211e')
 
     variant('gui',    default=True, description='Enable VisIt\'s GUI')
+    variant('adios2', default=False, description='Enable ADIOS2 file format')
     variant('hdf5',   default=True, description='Enable HDF5 file format')
     variant('silo',   default=True, description='Enable Silo file format')
     variant('python', default=True, description='Enable Python support')
     variant('mpi',    default=True, description='Enable parallel engine')
 
+    patch('spack-changes.patch')
+    patch('nonframework-qwt.patch', when='^qt~framework platform=darwin')
+
     depends_on('cmake@3.0:', type='build')
-    depends_on('vtk@6.1.0~opengl2~mpi')
-    depends_on('qt@4.8.6', when='+gui')
+    depends_on('vtk@8.1.0:+opengl2', when='@3.0:3.0.1')
+    depends_on('vtk@6.1.0~opengl2', when='@:2.999')
+    depends_on('vtk+python', when='+python @3.0:')
+    depends_on('vtk~mpi')
+    depends_on('vtk+qt', when='+gui')
+    depends_on('qt@4.8.6:4.999', when='+gui @:2.999')
+    depends_on('qt@5.10:', when='+gui @3.0:')
     depends_on('qwt', when='+gui')
-    depends_on('python', when='+python')
+    depends_on('python@2.6:2.8', when='+python')
```

Comment:
Any idea when visit will support Python 3?

Reply:
I haven't a clue -- that question would be better addressed to the primary VisIt developers (@chuckatkins ?)